deploy/olm-catalog/ibm-mongodb-operator/ibm-mongodb-operator.package.yaml

```diff
@@ -2,4 +2,4 @@ channels:
 - currentCSV: ibm-mongodb-operator.v0.0.1
   name: alpha
 defaultChannel: alpha
-packageName: ibm-mongodb-operator
+packageName: ibm-mongodb
```

Comment:
can we use `ibm-mongodb-operator-app` to keep consist with others?

https://quay.io/application/

Reply:
sure